scripts/mock-rapid-result.json

```diff
@@ -1,37 +0,0 @@
-[
```

Comment:
@JiangLong2019 can you make this change as part of pr https://github.com/aemsites/icicidirect/pull/226 ?